File: tests/mongo_connect.py
#!/usr/bin/env python3

from pymongo import MongoClient
from pprint import pprint


if __name__ == "__main__":
    # Need updated customer password
    client = MongoClient("mongodb+srv://customer:glcgQXd4prd1C3yk@innoventory-vvoxp.azure.mongodb.net/test?retryWrites=true&w=majority")
    db = client.admin

    serverStatusResult = db.command("serverStatus")
    pprint(serverStatusResult)

File: tests/mongo_connect.js
var MongoClient = require('mongodb').MongoClient;
var url = "mongodb+srv://<user>:<pass>@innoventory-vvoxp.azure.mongodb.net/test?retryWrites=true&w=majority";

MongoClient.connect(url, function(err, db) {
  if (err) throw err;
  var dbo = db.db("Innoventory");
  var query = { item: "Clam Nectar" };
  dbo.collection("Grocery").find(query).toArray(function(err, result) {
    if (err) throw err;
    console.log(result);
    db.close();
  });
});